man/systemctl.xml

```diff
@@ -1063,8 +1063,10 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
       <para><command>systemd</command> supports an environment block that is passed to processes the manager
```

Comment:
i am not convinced this is really a great idea. I'd not blanekt allow this. Maybe supporting ESC is OK. but ^H is just weird, because it allows to hide stuff. I am not sure why we should allow passing that to supposedly well-defined service environments...

The reference to PS1 in the commit msg should go. bash documents that escape and friends are encoded as `\e` last time I looked. If once encodes it as literal escape anyway instead, then maybe one deserves us complaining.

Similar: what about $LESS? where whould I use cc chars there?

I have the suspicion the termcap stuff supports proper escaping on the parser level too, but given that it is undocumented we'll never know